shells/pipes-shell/BUILD

```diff
@@ -6,12 +6,24 @@ run_in_repo(
     name = "pipes_shell_web",
     srcs = glob(["**"]),
     outs = ["pipes_shell_web_dist"],
-    cmd = "cd shells/pipes-shell/web/deploy && " +
+    cmd = "set OLDPWD=$PWD && cd $PWD/$(dirname $(location //shells/pipes-shell:web_deploy_bin)) && " +
           "./deploy.sh && " +
-          "cp -r dist {OUT}",
+          "cp -r dist $OLDPWD/{OUT}",
     progress_message = "Webpacking pipes-shell",
     visibility = ["//visibility:public"],
     deps = [
-        "//tools:sigh_webpack",
+        "//tools:sigh_bin", 
+        ":web_deploy",
+        ":web_deploy_bin",
     ],
 )
+
+filegroup(
+    name = "web_deploy",
+    srcs = glob(["web/deploy/**"]),
```

Comment:
Doesn't seem like there's a need for this. There's already a `srcs = glob(["**"])` line above, so these files are already being pulled in

Reply:
$location expansion doesn't work on a glob that has more than one file, so you need a target that has just one file, however you're right in that it is being double included, so the target above should exclude deploy.sh